README.md

```diff
@@ -967,6 +967,36 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='spaces-in-and-around-comments'></a>(<a href='#spaces-in-and-around-comments'>link</a>) Include spaces before and after comments [![SwiftFormat: spaceAroundComments](https://img.shields.io/badge/SwiftFormat-spaceAroundComments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#spaceAroundComments) [![SwiftFormat: spaceInsideComments](https://img.shields.io/badge/SwiftFormat-spaceInsideComments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#spaceInsideComments)
```

Comment:
Might be worth having a small clarification that a newline is fine for after comments as well?

Reply:
Maybe _"include spaces or newlines before or after comments"_ would work here

Reply:
good idea

Reply:
This is an improvement. I do wonder if we do another revision where we incorporate when it's OK to have a newline instead of a space, per Eric's comment.

Reply:
Updated it to say "Include spaces or newlines before and after comment delimiters"

Reply:
It's worth noting that in some places we require a space and in others we require a newline but I can get onboard with the idea that we don't need to be that precise in the prose.